modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java

```diff
@@ -339,17 +342,21 @@ private String _getAPIURL(
 
 		Map<String, Object> properties = fdsEntryObjectEntry.getProperties();
 
+		String restApplication = String.valueOf(
+			properties.get("restApplication"));
+
 		StringBundler sb = new StringBundler(3);
 
 		sb.append("/o");
 		sb.append(
-			StringUtil.replaceLast(
-				String.valueOf(properties.get("restApplication")), "/v1.0",
-				StringPool.BLANK));
+			StringUtil.replaceLast(restApplication, "/v1.0", StringPool.BLANK));
 		sb.append(String.valueOf(properties.get("restEndpoint")));
 
-		return _interpolateURL(
-			_getNestedFields(sb.toString(), fdsFieldObjectEntries),
+		return _resolveParameters(
+			_interpolateURL(
+				_getNestedFields(sb.toString(), fdsFieldObjectEntries),
+				httpServletRequest),
+			restApplication, String.valueOf(properties.get("restSchema")),
```

Comment:
I would say so. I would use this epic https://liferay.atlassian.net/browse/LPD-25230 as the LPD number for it, as I think we should create a story under it to track it.

Reply:
@herrardo Do we have a title and a description for the FF? Should I do it myself or should wait for someone else to enclose the functionality within the FF?

Reply:
I believe it can be a dev FF, that has no title and description. This seems quite a technical update to be customer-facing.  @herrardo may correct me on that one.